core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -637,6 +637,30 @@ public void testCompleteCreateTable() {
         .isEqualTo(UUID.fromString(((BaseTable) table).operations().current().uuid()));
   }
 
+  @Test
+  public void testDefaultTableProperties() {
```

Comment:
I would just create a separate PR to fix this for tables, because the scope of this PR is really about adding support for view default properties

Reply:
https://github.com/apache/iceberg/pull/11646